pimd/pim_cmd.c

```diff
@@ -795,15 +795,27 @@ static void igmp_show_interfaces_single(struct pim_instance *pim,
 	}
 }
 
-static void igmp_show_interface_join(struct pim_instance *pim, struct vty *vty)
+static void igmp_show_interface_join(struct pim_instance *pim,
+					struct vty *vty,
```

Comment:
Let's take a few minutes and get things aligned to our coding standard.  Please use clang-format.